modules/mobkoiIdSystem.js

```diff
@@ -12,14 +12,14 @@ import { logError, logInfo, deepAccess, insertUserSyncIframe } from '../src/util
 
 const GVL_ID = 898;
 const MODULE_NAME = 'mobkoiId';
-export const PROD_AD_SERVER_BASE_URL = 'https://adserver.maximus.mobkoi.com';
+export const PROD_INTEGRATION_BASE_URL = 'https://integration.maximus.mobkoi.com';
```

Comment:
hi, this adapter appears to have a memory leak from not removing its event listeners